packager/signer/signer.go

```diff
@@ -318,6 +329,14 @@ func (this *Signer) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
 		}
 	}()
 
+	// According to the check above httpErr is nil, i.e. the gateway request did
+	// succeed. Let Prometheus observe the gateway request along with the response code.
+	// Note that if httpErr was not nil, ServeHTTP wouldn't have reached the code
+	// below. Instead it would let mux's promRequestsTotal observe the non-gateway
+	// request (along with the response code) - by calling LogAndRespond with resp
+	// that mux have decorated with a promRequestsTotal InstrumentHandler.
```

Comment:
Maybe mention specifically that this will be http.StatusBadGateway (502), which gives the reader something to Ctrl-F for in this file, and the key to look for in the promRequestsTotal metric.

Reply:
Done. It can also be 500, so I used 502 as a probable example.